include/inexor/vulkan-renderer/frame_graph.hpp

```diff
@@ -251,7 +261,7 @@ class PhysicalBuffer : public PhysicalResource {
     friend FrameGraph;
 
 private:
-    VkBuffer m_buffer{VK_NULL_HANDLE};
+    VkBuffer m_buffer{nullptr};
```

Comment:
Hmm, I'm not sure about this change. I think that `VK_NULL_HANDLE` should still be used for vulkan objects, since it's vulkan's way of representing null and `VK_NULL_HANDLE` isn't guaranteed to be the same as `nullptr`. Although, if it causes a static analysis warning (since it's just a dumb macro), then I'm not too sure.

Reply:
I will undo this.

Reply:
Done.